### PR TITLE
[0.63] Backport codegen test fix for build on new agents 

### PR DESCRIPTION
**Pull request by AgneLukoseviciute**
Backporting #7398 

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/7428)

### CODE REVIEW

**Comment by AgneLukoseviciute** [comment]
Should help with CI Build for #7418

**Review by asklar — APPROVED** [review]

File: change/react-native-windows-3e2778f7-39fc-4d9a-9fc9-3b8c68a0975f.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "[0.63] codegen test fix for build on new agents",
```
Comment (NickGerleman) [inline change/react-native-windows-3e2778f7-39fc-4d9a-9fc9-3b8c68a0975f.json:3]: nit: These show up in release notes, so we usually avoid prefixing with a version. `yarn change` will let you type in anything for these, so it's sometimes worth giving a more "user-facing" description than what would normally be in a commit message.

**Comment by NickGerleman** [comment]
There are different breaks in 0.63 right now, so we will need to admin merge this, since CI will not pass yet. Either Alex or I can do that.

File: change/react-native-windows-3e2778f7-39fc-4d9a-9fc9-3b8c68a0975f.json (hunk 1 above)
Comment (NickGerleman) [reply change/react-native-windows-3e2778f7-39fc-4d9a-9fc9-3b8c68a0975f.json:3]: This is an example of what these look like, if you go to the "GitHub" releases view, or receive notifications from the repo. 
image

**Review by NickGerleman — APPROVED** [review]

**Comment by asklar** [comment]
yeah it's missing the certificate fix. See commit 1615a2432235c267005d044ef5ebf4e3a74cea7d - PR #7399

**Pull request merged**